WBooks/Screens/TabBarController.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  TabBarController.swift
+//  WBooks
+//
+//  Created by Nacho 2 on 24/01/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class TabBarController: UITabBarController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tabBar.barTintColor = .white
+        setupTabs()
+    }
+    
+    func setupTabs() {
+        let main = WBooksNavigationController(rootViewController: MainViewController())
```

Comment:
You are going to add more tabs and functionalities.. so maybe `MainViewController` should be renamed to `LibraryViewController` ... or something like that.. (just suggestion)

Reply:
nice! i'll keep in mind for the next card (MVVM refactor 😄 )